### DISCUSSION TITLE
Is there a way to print the exported text with formatting in the text itself?

### DISCUSSION THREAD

**Discussion by Ryang20718** [mindee/doctr#1600] [Q&A]
I.e, is there a way to print the resulting text in plaintext with formatting and spacing kept?

synthetic_pages = result.synthesize()
plt.imshow(synthetic_pages[0]); plt.axis('off'); plt.show()

**Comment by felixdittrich92** [comment]
Hi @Ryang20718 👋
You can use:
`plain = result.render()`

We provide several options to customize the output:
https://mindee.github.io/doctr/using_doctr/using_models.html#two-stage-approaches

**Reply by Ryang20718** [reply]
Thanks for the quick response @felixdittrich92!

I was wondering whether it's possible to print text (in the same row on a pdf) in the same way via `result.render()`

![po1](https://github.com/mindee/doctr/assets/31294356/46931a21-9754-47d1-befe-b03ece870f48)
I.e some file like the above generates 



```
<left tabular col>

<right tabular col>

```

rather than

```
<left tabular col> <right tabular col>
```

**Reply by felixdittrich92** [reply]
Hi @Ryang20718 👋
Have you tried to disable the block resolving with `resolve_blocks=False` ? :)

**Reply by Ryang20718** [reply]
yeah, originally tried that, but same issue. not a big deal! just wanted to see if there's a preexisting function I was missing. if not all good :) thanks Felix
